tests/travis-ci-apache.conf

```diff
@@ -0,0 +1,26 @@
+<VirtualHost *:80>
+	ServerAdmin webmaster@localhost
+	DocumentRoot %TRAVIS_BUILD_DIR%
+
+	<Directory />
+		Options FollowSymLinks
+		AllowOverride All
+	</Directory>
+
+	<Directory "%TRAVIS_BUILD_DIR%">
+		Options FollowSymLinks MultiViews ExecCGI
+		AllowOverride All
+		Order deny,allow
+		Allow from all
+	</Directory>
+
+	# Wire up Apache to use Travis CI's php-fpm.
+	<IfModule mod_fastcgi.c>
+		AddHandler php5-fcgi .php
```

Comment:
Actually I just copied this file from one I worked a long time ago for b2b
I'm not sure if there is a `php7-fcgi` binary, it would take some testing to see if it works, but `php5-fcgi` is working for both